tests/e2e/support/commands.js

```diff
@@ -75,7 +75,12 @@ Cypress.Commands.add('setupGameAsP0', (alreadyAuthenticated = false, isRanked =
     if (!alreadyAuthenticated) {
       cy.signupOpponent(opponentOne);
     }
-    cy.subscribeOpponent(gameSummary.gameId);
+    try {
+      cy.subscribeOpponent(gameSummary.gameId);
+    } catch {
+      cy.recoverSessionOpponent(opponentOne);
+      cy.subscribeOpponent(gameSummary.gameId);
+    }
```

Comment:
So the problem we're seeing in CI is sometimes the request to subscribe fails because the `io` instance registers as unauthenticated. This feels like a kludge of a fix because it's 1-time operation (though we probably don't want to loop it), so idk maybe it's fine? 

@seriouslysean what do you think? If we're gonna do it, I can at least roll it out into `setupGameAsP1()`

Reply:
Can you explain to me a little more about when it occurs? Is it every time, sometimes, unpredictable? Marine some sort of ab race condition where it’s connecting too soon?

Reply:
At this point it seems to happen once per CI run in a random one-off spec, mainly sixes and aces

Reply:
I wrote up #455 to investigate the root cause

Reply:
As long as this doesn't cause other problems with it, I'm fine with this for now. It might be better to have a poll try until it succeeds for up to 10 seconds but if this passes and is a test only then whatever. 👍🏼